front/src/applications/stdcm/components/StdcmHelpModule/types.ts

```diff
@@ -0,0 +1,38 @@
+export type Section = {
+  id: string;
+  title: string;
+  content?: Content[];
+  subSections?: Omit<Section, 'subSection'>[];
+};
+
+export type Content = ListContent | TextContent | FAQContent | GlossaryContent;
+
+export type ListContent = {
+  type: 'list';
+  items: Record<string, string>[];
```

Comment:
It seems like the only accepted field is `value`, so this can be tightened to

```ts
items: { value: string }[];
```

But at that point we might as well just use strings without the wrapper?

```ts
items: string[];
```

Reply:
agreed with the first point, but for the second i think i would just make the json less consistent and readable, because all the other type of item works with key: value